packages/core/admin/admin/src/content-manager/components/RelationInput/RelationInput.js

```diff
@@ -38,12 +38,13 @@ export const LinkEllipsis = styled(Link)`
 
 export const DisconnectButton = styled.button`
   svg path {
-    fill: ${({ theme }) => theme.colors.neutral500};
+    fill: ${({ theme, disabled }) =>
+      !disabled ? theme.colors.neutral500 : theme.colors.neutral600};
```

Comment:
You should do the opposite: `disabled ?  theme.colors.neutral60: theme.colors.neutral500` it's easier to understand

Reply:
missed your comments 😄
I agree, will do!